### ISSUE TITLE
is it possible to install older versoins?

### ISSUE THREAD

**Issue by eldritchFyou** [cheat-engine/cheat-engine#1690]
I wanted to download one of the 6. versions but I can't find the installer is it still possible to download it or is it no longer possible?

**Comment by Skip75** [comment]
Yes, find them here : https://github.com/cheat-engine/cheat-engine/releases
